build.zig

```diff
@@ -118,34 +134,56 @@ fn build_examples(b: *Build, webui: *Compile) !void {
 
     const examples_path = (if (zig_ver < 12) (Build.LazyPath{ .path = "examples/C" }) else b.path("examples/C")).getPath(b);
     var examples_dir = if (zig_ver < 12)
-        try std.fs.openIterableDirAbsolute(examples_path, .{})
```

Comment:
Is there any difference between these two usages?
Is there any relevant documentation or discussion notes?

Reply:
The Zig core team recommends avoiding absolute paths in all situations, as numerous issues have been filed in the past related to absolute file paths used in very unusual mounting/filesystem environments. 

As such, Zig best practice is to pin all filesystem operations on `cwd` directory base. Absolute file paths even when operated from `cwd` directory will still resolve successfully, so there is no reason not to follow this best practice.

As for documentation/discussion notes, here's one relevant curently open issue:

Reply:
I see
This is not the same as my past experience using c